### DISCUSSION TITLE
Support for GGUF models?

### DISCUSSION THREAD

**Discussion by aalyousfi** [outlines-dev/outlines#427] [Q&A] [answered]
Hi there,

Is there a plan to add support to GGUF models (from llama.cpp)? They are very popular and it would be very convenient if we can use them with Outlines.

Thank you!

**Comment by doomgrave** [comment]
UP for Llama-CPP support!

**Comment by rlouf** [comment]
This is being implemented in #486

**Comment by rlouf** [comment, marked as answer]
It is now available in the latest version. See the [documentation](https://outlines-dev.github.io/outlines/reference/models/llamacpp/).

**Reply by doomgrave** [reply]
This is a great upgrade!!!
Going to test ASAP

**Reply by aalyousfi** [reply]
Thank you!

**Reply by rlouf** [reply]
Thank @dtiarks !
